java/ql/lib/ext/javax.servlet.http.model.yml

```diff
@@ -26,6 +26,7 @@ extensions:
       - ["javax.servlet.http", "HttpServletResponse", False, "addHeader", "", "", "Argument[0..1]", "response-splitting", "manual"]
       - ["javax.servlet.http", "HttpServletResponse", False, "sendError", "(int,String)", "", "Argument[1]", "information-leak", "manual"]
       - ["javax.servlet.http", "HttpServletResponse", False, "setHeader", "", "", "Argument[0..1]", "response-splitting", "manual"]
+      - ["javax.servlet.http", "HttpServletResponse", True, "sendRedirect", "(String)", "", "Argument[0]", "url-redirection", "ai-manual"]
```

Comment:
This is already handled in regular CodeQL: https://github.com/github/codeql/blob/67f8bcce440017cb8bd63656538c4fcec00d3202/java/ql/lib/semmle/code/java/security/UrlRedirect.qll#L19

I'm not sure what to do in these cases. We could merge it and accept the duplicate until we clean up, merge it only if we also clean it up in the same PR, or not merge it at all.

Reply:
cd765e7 I've commented the model out, which is what we've done previously in similar situations